packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -30,6 +30,8 @@ export const defaultLayouts = {
 	[ LAYOUT_TABLE ]: {
 		layout: {
 			primaryField: 'title',
+			hierarchical: 'parent',
```

Comment:
Is `hierarchical` prop needed and/or used? I guess it would be needed if we had client side handling for it, right?